pymongoexplain/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '0.1.0.dev0'
+__version__ = '0.1.0'
```

Comment:
Are cutting a 0.1.0 release? I thought it will be a 1.0.0 release.

Reply:
I was confused about that. I will change it to 1.0

Reply:
> Please note the following while you are merging this:
> 
> The commit message should be something like `BUMP x.y.z` (see e.g. [mongodb/mongo-python-driver@63574b9](https://github.com/mongodb/mongo-python-driver/commit/63574b93603cda2c87fd99f1b90033970380b72b))
> 
> Once that commit has been tagged, please open another PR which bumps this version to the next minor version and adds the `dev0` tag back : `1.1.0dev0`

Will do